test/network_constructors.jl

```diff
@@ -0,0 +1,216 @@
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.EqualTo{Float64}) == 264
+
+    JuMP.@objective(ps_model.JuMPmodel, Min, AffExpr(0))
+    JuMP.optimize!(ps_model.JuMPmodel)
+
+    @test termination_status(ps_model.JuMPmodel) == MOI.OPTIMAL
+end
+
+ @testset "PTDF ArgumentError" begin
+    ps_model = PSI._ps_model_init(sys5b, GLPK_optimizer, PM.AbstractPowerFormulation, sys5b.time_periods)  
+    PSI.construct_device!(ps_model, PSY.ThermalGen, PSI.ThermalDispatch, PSI.StandardPTDFForm, sys5b, time_range);
+    PSI.construct_device!(ps_model, PSY.PowerLoad, PSI.StaticPowerLoad, PSI.StandardPTDFForm, sys5b, time_range);
+    @test_throws ArgumentError PSI.construct_network!(ps_model, PSI.StandardPTDFForm, sys5b, time_range)
+end
+
+@testset "testing DC-PF network construction" begin
+    ps_model = PSI._ps_model_init(sys5b, GLPK_optimizer, PM.AbstractPowerFormulation, sys5b.time_periods)  
+    PSI.construct_device!(ps_model, PSY.ThermalGen, PSI.ThermalDispatch, PM.DCPlosslessForm, sys5b, time_range);
+    PSI.construct_device!(ps_model, PSY.PowerLoad, PSI.StaticPowerLoad, PM.DCPlosslessForm, sys5b, time_range);
+    PSI.construct_network!(ps_model, PM.DCPlosslessForm, sys5b, time_range);
+    @test JuMP.num_variables(ps_model.JuMPmodel) == 384
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.LessThan{Float64}) == 144
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.GreaterThan{Float64}) == 144
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.EqualTo{Float64}) == 288
+
+    JuMP.@objective(ps_model.JuMPmodel, Min, AffExpr(0))
+    JuMP.optimize!(ps_model.JuMPmodel)
+
+    @test termination_status(ps_model.JuMPmodel) == MOI.OPTIMAL
+end
+
+@testset  "testing AC-PF network construction" begin
+    ps_model = PSI._ps_model_init(sys5b, ipopt_optimizer, PM.AbstractPowerFormulation, sys5b.time_periods)  
+    PSI.construct_device!(ps_model, PSY.ThermalGen, PSI.ThermalDispatch, PM.StandardACPForm, sys5b, time_range);
+    PSI.construct_device!(ps_model, PSY.PowerLoad, PSI.StaticPowerLoad, PM.StandardACPForm, sys5b, time_range);
+    PSI.construct_network!(ps_model, PM.StandardACPForm, sys5b, time_range);
+    @test JuMP.num_variables(ps_model.JuMPmodel) == 1056
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.LessThan{Float64}) == 144
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.GreaterThan{Float64}) == 144
+    @test JuMP.num_constraints(ps_model.JuMPmodel,GenericAffExpr{Float64,VariableRef},MOI.EqualTo{Float64}) == 264
+
+    JuMP.@objective(ps_model.JuMPmodel, Min, AffExpr(0))
+    JuMP.optimize!(ps_model.JuMPmodel)
+
+    @test termination_status(ps_model.JuMPmodel) in [MOI.OPTIMAL, MOI.LOCALLY_SOLVED]
+end
+
+#=
```

Comment:
Suggest adding an explanatory note any time a large block of code like this is commented out.

Reply:
Ok, I should actually move some of these tests to another file actually

Reply:
resolved via 2a7f61a